### DISCUSSION TITLE
Pull API Keys & Relevant Info from CIDs

### DISCUSSION THREAD

**Discussion by ProGritz** [CrowdStrike/psfalcon#299] [Ideas]
I'm not sure if it's already been considered since I think there is an old API URL that did just this. But is there any plans for having PSFalcon or the API itself allow admins to pull info related to API keys?

I'm looking to automate the recycling of API keys based on the creation date.

**Comment by bk-cs** [comment]
There is no API available that will allow access to view or manage API credentials.
